### PR TITLE
Docs: Auth Auto-Detect

### PR DESCRIPTION
**Pull request by psiinon**
Improvement suggestions gratefully received!

### CODE REVIEW

**Review by kingthorin — APPROVED** [review]

File: site/content/docs/authentication/authentication-methods.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+title: "Authentication Methods"
+tags: 
+- authentication
+- guide
+type: page
+---
+Authentication Methods are the means by which ZAP actually authenticated to a web app.
```
Comment (thc202) [inline site/content/docs/authentication/authentication-methods.md:8]: `authenticates`?

File: site/content/docs/authentication/auto-detection.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+If they work for your apps then it will make your life considerably easier, so this is the best place for you to start.
+
+### Authentication Tester Dialog
+
+The [Authentication Tester Dialog](/blog/2023-05-23-authentication-tester/) is a quick and easy way to check if ZAP can 
+automatically detect all of the information it needs - you only need to provide the URL of your login page and valid credentials.
+
+If everything passes then auto-detection is probably going to be your best option.
+
+If if fails then the dialog will collect sanitised diagnostics which you can safely share with us in order to help identify what is going wrong.
```
Comment (thc202) [inline site/content/docs/authentication/auto-detection.md:18]: `If it`

File: site/content/docs/authentication/auto-detection.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+### Auto Detection in the Desktop
+
+The Authentication Tester Dialog will automatically create a new [Context](/docs/desktop/start/features/contexts/) for you (by default "Authentication Test").
+This will be updated with the details that ZAP detects.
+
+You can also set up authentication auto-detection manually in the ZAP Desktop by:
+
+* Adding your app to a [Context](/docs/desktop/start/features/contexts/)
+* Setting the [Authentication](/docs/desktop/ui/dialogs/session/context-auth/#authentication-method) to [Browser-based Authentication]()
```
Comment (thc202) [inline site/content/docs/authentication/auto-detection.md:28]: Missing path in the second link?

**Comment by psiinon** [comment]
Thanks - fixed those 😁

**Review by thc202 — APPROVED** [review]

**Pull request merged**